tests/test_bundle.py

```diff
@@ -828,6 +828,7 @@ def test_is_directory(self):
 
     def test_is_not_executable(self):
         with tempfile.NamedTemporaryFile() as tmpfile:
+            # assert os.access(tmpfile.name, os.X_OK) == False #for windows this fails, linux this passes
```

Comment:
Not sure what to do with this test: inside the context manager of a temp file in MacOS/Linux , `os.access(tmpfile.name, os.X_OK) == False`, which triggers the RSConnectError. 

However, on Windows, `os.access(tmpfile.name, os.X_OK) == True`, so this test fails.

Reply:
Looking at the [comp.lang.python google group](https://groups.google.com/g/comp.lang.python/c/FCiVzouJt5s) , it seems like, on Windows, `os.access()` will generally return True if the path can be accessed at all.

I tried to update with `os.chmod(tmpfilename, 000)` and still could not force the RSConnectException 😩.

NOTE: the discussion is from 2007, but it does seem that the behavior is still the same.